src/main/java/com/github/onsdigital/perkin/transform/TransformEngine.java

```diff
@@ -24,77 +20,42 @@
 
     private static TransformEngine INSTANCE = new TransformEngine();
 
-    private SurveyParser parser = new SurveyParser();
+    private SdxValidate validate = new SdxValidate();
+    private SdxStore store = new SdxStore();
     private Decrypt decrypt;
 
     private Audit audit = Audit.getInstance();
-    private TemplateLoader loader = TemplateLoader.getInstance();
-
-    //private List<Transformer> transformers;
-
-    private FtpPublisher publisher = new FtpPublisher();
-
-    private NumberService batchNumberService = new NumberService("batch", 30000, 39999);
-    private NumberService sequenceNumberService = new NumberService("sequence", 1000, 99999);
-    private NumberService scanNumberService = new NumberService("scan", 1, 999999999);
 
     private TransformEngine() {
         //use getInstance()
-
-        Runtime.getRuntime().addShutdownHook(new Thread() {
-            public void run() {
-                log.info("SHUTDOWN|storing sequence numbers...");
-                batchNumberService.save();
-                sequenceNumberService.save();
-                scanNumberService.save();
-            }
-        });
     }
 
     public static TransformEngine getInstance() {
         return INSTANCE;
     }
 
-    public List<DataFile> transform(final String data) throws TransformException, InterruptedException, ExecutionException {
+    public void transform(final String data) throws TransformException, InterruptedException, ExecutionException {
 
         Timer timer = new Timer("survey.process.");
 
-        ExecutorService taskExecutor = Executors.newFixedThreadPool(4);
-        CountDownLatch latch = new CountDownLatch(4);
-
         try {
             decrypt = new Decrypt(data);
             String json = decrypt.getDecrypted();
-            Survey survey = parser.parse(json);
 
-            TransformContext context = createTransformContext(survey);
+            //TODO will this throw an Exception or use a return value?
+            //{"valid": True} 200
+            //{"valid": False} 400?
+            Response<Result> response = validate.validate(json);
```

Comment:
Can see the todos about response codes here - invalid json will respond with 400, so if that can be correctly handled that would be great.